pkg/gui/presentation/commits.go

```diff
@@ -99,6 +101,14 @@ func GetCommitListDisplayStrings(
 		getGraphLine = func(idx int) string { return "" }
 	}
 
+	branchHeadsToVisualize := lo.FilterMap(branches,
+		func(b *models.Branch, index int) (string, bool) {
+			return b.CommitHash,
```

Comment:
Each of these conditions would be good to explain in a comment so that we know what the underlying goal is. For example it's not clear to me why we exclude the start bisect commit. I also think this warrants some documentation somewhere so that it's easy to explain the logic when a user asks about it.

Reply:
Some of this logic has now moved to `local_commits_context.go`; I added some comments both here and there.

Reply:
Those comments are great!